runtime/stdlib/contracts/crypto.cdc

```diff
@@ -174,16 +160,14 @@ pub contract Crypto {
 
     priv let domainSeparationTagUser: String
 
-    priv let signatureVerifier: {SignatureVerifier}
     priv let hasher: {Hasher}
 
-    init(signatureVerifier: {SignatureVerifier}, hasher: {Hasher}) {
+    init(hasher: {Hasher}) {
 
-        self.signatureVerifier = signatureVerifier
         self.hasher = hasher
 
         // Initialize constants
 
-        self.domainSeparationTagUser = "user"
+        self.domainSeparationTagUser = "FLOW-V0.0-user"
```

Comment:
@turbolent @tarakby the tag should be left padded here right? The previous `user` tag was replaced by a left padded `FLOW-V0.0-user` tag.

I'm asking because I'm trying to upgrade Cadence in flow-go but I'm getting errors in tests that cover this. I traced the errors back to this.